### ISSUE TITLE
Squirrel uninstall event not being called

### ISSUE THREAD

**Issue by btoda** [Squirrel/Squirrel.Windows#747]
I have this issue when uninstalling a squirrel aware app from Windows 10(add-remove features) - probably other windows os too.
The squirrel uninstall event is not being called.
I have a log that writes into a file when the app is run. I have these lines for the first run:
[12:47:28] - Application starting...
[12:47:28] - Arguments:C:\Users\Bogdan\AppData\Local\MyApp\app-2.1.0.53\MyApp.exe, --squirrel-firstrun

However when I uninstall the app, I have nothing there. So i suspect update.exe does not run the app at all when uninstalling?

Any ideas? I mean I could hijack the uninstall values in registry and put my own, but I would prefer to do it legit. Any idea if I am doing something wrong here?

**Comment by junjunruan** [comment]
Hi,
how did you create a log file?
I have the squirrel events like "--squirrel-install", "--squirrel-uninstall", "--squirrel-updated" in my code, and I use fs to write logging to a file. In my case, I can only see logs about uninstall event, and didn't see other events.

**Comment by the-dansan** [comment]
Yeah, I am having this issue as well. Uninstall works fine if I call the app directly with the arguments, but if I rely on the registry value, it fails.

It appears that since we have non-standard installation stuff (e.g., directory structure) squirrel doesn't know where to go to find my exe to call.
